common/services/prismic/fetch-links.js

```diff
@@ -35,6 +35,13 @@ export const collectionVenuesFields = [
   'collection-venue.sunday',
   'collection-venue.modifiedDayOpeningTimes',
 ];
+
+export const mediaObjectFields = [
+  'media_object.title',
```

Comment:
Any reason we're going with `_` over `-`?

Reply:
nope.. this was generated in prismic. which got me caught out..

Reply:
Could we change it?

Reply:
Will change this.. i had this originally media-object and then found out that prismic generated media_object .. ill try and change this

Reply:
<img width="1316" alt="Screenshot 2020-10-27 at 09 21 54" src="https://user-images.githubusercontent.com/9095365/97281887-f05ea580-1835-11eb-91e3-6690a49a282e.png">

Reply:
You need to change it on creation.

Reply:
i see .. and then after rename the name of the media object . thanks